common/test/acceptance/tests/lms/test_programs.py

```diff
@@ -102,6 +102,33 @@ def test_enrollments_and_programs(self):
         self.assertTrue(self.listing_page.are_cards_present)
 
 
+class ProgramMarketingPageTest(ProgramPageBase):
+    """Verify user-facing behavior of the program marketing page."""
+    def setUp(self):
+        super(ProgramMarketingPageTest, self).setUp()
+
+        self.marketing_page = ProgramMarketingPage(self.browser)
+
+        self.program = self.create_program()
+        self.program['uuid'] = self.marketing_page.program_uuid
+
+    def test_program_course_cards(self):
+        """
+        Verify that course cards appear when a user visits the marketing page.
+        """
+        self.auth()
+
+        self.stub_catalog_api(programs=[self.program])
+        self.cache_programs()
+
+        self.marketing_page.visit()
+
+        self.assertTrue(self.marketing_page.are_course_cards_present)
```

Comment:
We should already have a unit test that can tell if you if the page will load. What's the value of checking if a few elements are on the page? Frankly, I'm not sure of the value of the test given that it doesn't check for actual program details.